### ISSUE TITLE
0.10.0 Context menu broken when more than 1 HOT instance is on page

### ISSUE THREAD

**Issue by trebuchetty** [handsontable/handsontable#1244]
You can see this on the context menu sample page. Open the console, then right-click on a cell on the 2nd or 3rd HOT shown and select an option from the context menu.
http://handsontable.com/demo/contextmenu.html
You'll notice that even the 1st HOT shown also throws this error, but the callback from the menu item is still executed, so it appears to work as expected.

![image](https://f.cloud.github.com/assets/2650592/1834648/6bf2608a-73e0-11e3-8d67-dca093a57cba.png)

When there are multiple HOT instances on the same page, the contextMenu's parent "instance" refers to the first HOT instance on the page, and not the HOT selected. This throws an error when looking for the selected cells as it thinks no cells are selected.

![image](https://f.cloud.github.com/assets/2650592/1834583/621cc38a-73df-11e3-903c-290301b677c2.png)

From the image above, that HOT is in a modal that's above another HOT. The "this" in the image is incorrectly referring to the HOT instance on the page and not the one in the modal that can be seen.

Basically any time there are multiple HOT's on a screen, the context menu will be broken, so I'd say this is a pretty important bug to fix (IMHO).
